Library/Models/SurveyDataRequest.cs

```diff
@@ -91,10 +91,25 @@ public class SurveyDataRequest
         public bool IncludeParaData { get; set; }
 
         /// <summary>
-        /// Include in the download the captured media files during the interviews
-        /// Previously: DownloadCapturedMedia
+        /// Include in the download the All captured Media Files during the interviews (AudioQuestion, VideoQuestion, PhotoQuestion, AudioSilentRecording)
```

Comment:
"Include all captured Media files ..." or "Include in the download all captured Media files ..."  would read a lot better.